src/main/java/org/apache/zeppelin/interpreter/Constants.java

```diff
@@ -23,4 +23,8 @@
   public static final String FS_S3A_IMPL = "fs.s3a.impl";
   public static final String SPARK_SQL_SHUFFLE_PARTITIONS = "spark.sql.shuffle.partitions";
   public static final String FS_S3A_CONNECTION_MAXIMUM = "fs.s3a.connection.maximum";
+  public static final String DEFAULT_USER_KEY = "default.user";
```

Comment:
Minor: Why two names for each of the properties? Can't we use the same (either user or default.user) property name?